lib/pages/chat_draft/draft_chat.dart

```diff
@@ -260,6 +259,40 @@ class DraftChatController extends State<DraftChat>
     );
   }
 
+  @override
+  void sendFileOnWebAction(
+    BuildContext context, {
+    Room? room,
+  }) async {
+    if (room == null) {}
+    final sendFileOnWebInteractor = getIt.get<SendFileOnWebInteractor>();
+    final result = await FilePicker.platform.pickFiles(
+      withData: true,
+    );
+    if (result == null || result.files.isEmpty) return;
+
+    _createRoom(
+      onRoomCreatedSuccess: (newRoom) {
+        sendFileOnWebInteractor.execute(
+          room: newRoom,
+          filePickerResult: result,
+        );
+      },
+    );
+  }
+
+  @override
+  Future<void> sendImages(
```

Comment:
why need to move over here, please make it less changes as little as possible to avoid error

Reply:
Because we need to create the room before sending the image

Reply:
no i means this method look exactly the old code